tests/ui/invalid_intopy_derive.rs

```diff
@@ -138,4 +138,27 @@ enum EnumStructIntoPyWith {
     },
 }
 
+#[derive(IntoPyObject, IntoPyObjectRef)]
+#[pyo3(transparent, rename_all = "camelCase")]
+struct StructTransparentRenameAll {
+    foo_bar: String,
+}
+
+#[derive(IntoPyObject, IntoPyObjectRef)]
+#[pyo3(rename_all = "camelCase")]
+struct StructTupleRenameAll(String, usize);
+
+#[derive(IntoPyObject, IntoPyObjectRef)]
+#[pyo3(rename_all = "camelCase")]
+enum EnumTransparentRenameAll {
+    #[pyo3(transparent)]
+    Variant { foo: String },
+}
+
+#[derive(IntoPyObject, IntoPyObjectRef)]
+#[pyo3(rename_all = "camelCase")]
+enum EnumTupleRenameAll {
+    Variant(String, usize),
+}
```

Comment:
Errors have not been emitted for these ones?

Reply:
Good catch! This was not working as I intended. The `rename_all` on the enum is now forwarded to each variant. I added additional tests to check it works as expected.